requirements.txt

```diff
@@ -21,3 +21,4 @@ psycopg2==2.7.1
 libsass==0.12.3
 IPython
 flask-shell-ipython
+uwsgi==2.0.15
```

Comment:
this shouldn't be in here. usually people install it using their distro's package manager